src/models/endpoint/participantEndpoint.js

```diff
@@ -0,0 +1,77 @@
+ Contributors
+ --------------
+ This is the official list of the Mojaloop project contributors for this file.
+ Names of the original copyright holders (individuals or organizations)
+ should be listed with a '*' in the first column. People who have
+ contributed from an organization can be listed under the organization
+ that actually holds the copyright for their contributions (see the
+ Gates Foundation organization for an example). Those individuals should have
+ their names indented and be marked with a '-'. Email address can be added
+ optionally within square brackets <email>.
+ * Gates Foundation
+ - Name Surname <name.surname@gatesfoundation.com>
+
+ - Shashikant Hirugade <shashikant.hirugade@modusbox.com>
+ --------------
+ ******/
+
+'use strict'
+
+const Logger = require('@mojaloop/central-services-shared').Logger
+const Config = require('../../lib/config')
+const Mustache = require('mustache')
+const request = require('request')
+
+/**
+* @function fetchEndpoints
+*
+ * @description This returns fetches the endpoints from a remote URI, so that it will can be cached in ml-api-adapter
+ *
+ * @param {string} fsp The fsp id
+ * @returns {object} endpointMap Returns the object containing the endpoints for given fsp id
+ */
+
+const fetchEndpoints = async (fsp) => {
+  const url = Mustache.render(Config.ENDPOINT_SOURCE_URL, { fsp })
+  const requestOptions = {
+    url,
+    method: 'get',
+    agentOptions: {
+      rejectUnauthorized: false
+    }
+  }
+  Logger.debug(`[fsp=${fsp}] ~ Model::Cache::fetchEndpoints := fetching the endpoints from the resource with options: ${JSON.stringify(requestOptions)}`)
+
+  return new Promise((resolve, reject) => {
+    return request(requestOptions, (error, response, body) => {
+      if (error) {
+        // throw error // this is not correct in the context of a Promise.
+        Logger.error(`[fsp=${fsp}] ~ Model::Cache::fetchEndpoints := Callback failed with error: ${error}, response: ${JSON.stringify(response)}`)
+        return reject(error)
+      }
+      Logger.info(`[fsp=${fsp}] ~ Model::Cache::fetchEndpoints := Callback successful with body: ${JSON.stringify(response.body)}`)
+      Logger.debug(`[fsp=${fsp}] ~ Model::Cache::fetchEndpoints := Callback successful with response: ${JSON.stringify(response)}`)
+      let endpoints = JSON.parse(body)
+      let endpointMap = {}
+
+      if (Array.isArray(endpoints)) {
+        endpoints.forEach(item => {
+          endpointMap[item.type] = item.value
+        })
+      }
+
+      Logger.debug(`[fsp=${fsp}] ~ Model::Cache::getEndpoints := Returning the endpoints: ${JSON.stringify(response)}`)
+      return resolve(endpointMap)
+    })
+  })
+}
+module.exports = {
+  fetchEndpoints
```

Comment:
Please align this to the function name as per the model in the Central-Ledger, which should just be `GetEndpoint`.